lib/content-services/src/lib/search/components/search-datetime-range/search-datetime-range.component.ts

```diff
@@ -133,8 +132,8 @@ export class SearchDatetimeRangeComponent implements SearchWidget, OnInit, OnDes
         if (isValid && this.id && this.context && this.settings && this.settings.field) {
             this.isActive = true;
 
-            const start = moment.utc(model.from).startOf('minute').format();
-            const end = moment.utc(model.to).endOf('minute').format();
+            const start = DateFnsUtils.formatDate(startOfMinute(new Date(model.from)), `yyyy-MM-dd'T'HH:mm:ss'Z'`);
```

Comment:
the idea of this code is to format datetime as UTC https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
so this can be expressed as 

```
startOfMinute(new Date(model.from)).toISOString()
```